### ISSUE TITLE
GPS will not connect.

### ISSUE THREAD

**Issue by chopper-2656** [tenable/pi_sniffer#6]
Just wondering if I am missing something in configuring the external /dev/ttyUSB0 serial GPS with the pi_sniffer. I modified the kismet.conf file to point to the correct the serial device. But no luck. 

I am using a GlobalSat BU-353 (SN-0218460)

**Comment by talmage** [comment]
Same question.  I'm using the U-Blox7 USB GPS recommended in README.md.  The kernel sees it at boot.  Pi_sniffer sees it but never gets sync.  None of the pi_sniffer output files have non-zero Lat or Lon.

**Comment by chopper-2656** [comment]
I found the issue with my GPS not being found. If you are using a USB device like I am (BU-353) then your device will be registered as /dev/ttyUSB0. I found that the display_handler.py has the GPS device hard coded as /dev/ttyAMC0 so unless your GPS is mapped to that device you will not have your GPS recognized. So just edit the display_handler.py and change AMC0 to USB0 or whatever you GPS is mapped to and then save the file and reboot.

Everything will work fine now.
